content/en/docs/tutorials/kubernetes-basics/_index.html

```diff
@@ -41,7 +41,7 @@ <h2>Kubernetes Basics</h2>
     <div class="row">
       <div class="col-md-9">
         <h2>What can Kubernetes do for you?</h2>
-        <p>With modern web services, users expect applications to be available 24/7, and developers expect to deploy new versions of those applications several times a day. Containerization helps package software to serve these goals, enabling applications to be released and updated in an easy and fast way without downtime. Kubernetes helps you make sure those containerized applications run where and when you want, and helps them find the resources and tools they need to work. Kubernetes is a production-ready, open source platform designed with Google's accumulated experience in container orchestration, combined with best-of-breed ideas from the community.</p>
+        <p>With modern web services, users expect applications to be available 24/7 and developers expect to deploy new versions of those applications several times a day. Containerization helps package software to serve these goals; enabling applications to be released and updated in an easy and fast way without downtime. Kubernetes helps you be sure the containerized applications run where and when you want, and helps the containers find the resources and tools they need to work. Kubernetes is a production-ready, open source platform designed with Google's accumulated experience in container orchestration, combined with best-of-breed ideas from the community.</p>
```

Comment:
And in this chunk, your proposed replacement is functionally equivalent to the original.